experiments/spbu_author/README.md

```
@@ -0,0 +1,125 @@
++           ├── works-csv-Lejnina.csv   
++           ├── works-csv-Krylatov.csv  
++           └── works-csv-Kizhaeva.csv  
++```
+
+Все CSV-файлы должны лежать **в папке data**.
+
+---
+
+## Зависимости
+
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)